frontend/kube/client.go

```diff
@@ -30,8 +33,30 @@ type Client struct {
 	ActionsClient actionsv1alpha1.ActionsV1alpha1Interface
 }
 
+func IsRunningInKubernetes() bool {
+	return os.Getenv("KUBERNETES_SERVICE_HOST") != ""
+}
+
+func getClientConfig(kc string) (*rest.Config, error) {
+	var kubeConfig *rest.Config
+	var err error
+
+	if IsRunningInKubernetes() {
+		kubeConfig, err = rest.InClusterConfig()
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		kubeConfig, err = clientcmd.BuildConfigFromFlags("", kc)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return kubeConfig, nil
+}
```

Comment:
nit: we can move this code to k8sutils instead of having it duplicated in cli and ui